pkg/render/manager.go

```diff
@@ -528,13 +537,20 @@ func (c *managerComponent) managerOAuth2EnvVars() []v1.EnvVar {
 	} else {
 		envs = []corev1.EnvVar{
 			{Name: "CNX_WEB_AUTHENTICATION_TYPE", Value: "OIDC"},
-			{Name: "CNX_WEB_OIDC_CLIENT_ID", Value: c.keyValidatorConfig.ClientID()}}
+			{Name: "CNX_WEB_OIDC_CLIENT_ID", Value: c.keyValidatorConfig.ClientID()},
+
+			// todo: remove this once manager correctly reads well-known-config from root of local domain
```

Comment:
Is this todo and the one below expected to come in before the next release?

Reply:
I'm not sure. It's not a blocker so UI team may punt to after CC release